src/psycop_feature_generation/text_models/preprocessing.py

```diff
@@ -24,7 +31,35 @@ def text_preprocessing(
     df[text_column_name] = (
         df[text_column_name]
         .str.lower()
-        .replace(regex_symbol_removal_and_stop_words, value="", regex=True)
+        .replace(regex_symbol_removal_and_stop_words, value="", regex=True)  # type: ignore
     )
 
     return df
+
+
+def text_preprocessing_pipeline(
+    split_name: Sequence[Literal["train", "val"]] = ["train", "val"],
```

Comment:
```suggestion
    split_names: Sequence[Literal["train", "val"]] = ["train", "val"],
```
F2 so it corrects below as well.